packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -379,6 +405,39 @@ export default {
             }
         },
 
+        /**
+         * Redirect to the payment page.
+         */
+        redirectToPayment () {
+            setTimeout(() => { // TODO: remove this when the order team handles this automatically.
+                window.location.assign(this.paymentPageUrl);
+            }, 1000);
+        },
+
+        /**
+         * Place the order.
+         */
+        async submitOrder () {
+            const data = {
+                basketId: this.basket.id,
+                applicationId: 7, // Responsive Web
+                customerNotes: {
+                    noteForRestaurant: this.userNote
+                },
+                applicationName: this.applicationName,
+                applicationVersion: '1',
+                referralState: 'None',
+                deviceId: '127.0.0.1', // TODO: TBC
```

Comment:
Still waiting for the Order team to see what we need to pass in here, or if we'll have to pass anything at all.